### PR TITLE
Add pcvSetVoltageEnabled and pcvGetVoltageEnabled

### PR DESCRIPTION
**Pull request by pixel-stuck**

### CODE REVIEW

**Comment by yellows8** [comment]
Formatting needs to match the other funcs.

File: nx/source/services/pcv.c [hunk 1]
```diff
@@ -104,3 +104,75 @@ Result pcvGetClockRate(PcvModule module, u32 *out_hz) {
 
     return rc;
 }
+
+Result pcvSetVoltageEnabled(bool state, u32 voltage) {
+    IpcCommand c;
+    ipcInitialize(&c);
+
+    struct {
+      u64 magic;
```
Comment (yellows8) [inline nx/source/services/pcv.c]: See other funcs whitespace.

File: nx/source/services/pcv.c [hunk 2]
```diff
@@ -104,3 +104,75 @@ Result pcvGetClockRate(PcvModule module, u32 *out_hz) {
...
+
+    raw->magic = SFCI_MAGIC;
+    raw->cmd_id = 8;
+    raw->state = state;
+    raw->voltage = voltage;
+
+    Result rc = serviceIpcDispatch(&g_pcvSrv);
+
+    if (R_SUCCEEDED(rc)) {
+      IpcParsedCommand r;
```
Comment (yellows8) [inline nx/source/services/pcv.c]: Use same whitespace as elsewhere.

File: nx/source/services/pcv.c [hunk 3]
```diff
@@ -104,3 +104,75 @@ Result pcvGetClockRate(PcvModule module, u32 *out_hz) {
...
 }
+
+Result pcvSetVoltageEnabled(bool state, u32 voltage) {
+    IpcCommand c;
+    ipcInitialize(&c);
+
+    struct {
+        u64 magic;
+        u64 cmd_id;
+        bool state;
```
Comment (yellows8) [inline nx/source/services/pcv.c]: u8

File: nx/source/services/pcv.c [hunk 4]
```diff
@@ -104,3 +104,75 @@ Result pcvGetClockRate(PcvModule module, u32 *out_hz) {
...
+    raw->voltage = voltage;
+
+    Result rc = serviceIpcDispatch(&g_pcvSrv);
+
+    if (R_SUCCEEDED(rc)) {
+        IpcParsedCommand r;
+        struct {
+            u64 magic;
+            u64 result;
+            bool isEnabled;
```
Comment (yellows8) [inline nx/source/services/pcv.c]: u8

File: nx/source/services/pcv.c [hunk 5]
```diff
@@ -104,3 +104,75 @@ Result pcvGetClockRate(PcvModule module, u32 *out_hz) {
...
+            u64 magic;
+            u64 result;
+            u8 isEnabled;
+        } *resp;
+
+        serviceIpcParse(&g_pcvSrv, &r, sizeof(*resp));
+        resp = r.Raw;
+
+        rc = resp->result;
+        *isEnabled = (bool)resp->isEnabled;
```
Comment (yellows8) [inline nx/source/services/pcv.c]: if (R_SUCCEEDED(rc) && isEnabled) ...

Comment (pixel-stuck) [reply nx/source/services/pcv.c]: Might want to do something similar with out_hz above too

Comment (yellows8) [reply nx/source/services/pcv.c]: sure

**Pull request merged**